src/pages/Profile.tsx

```diff
@@ -31,13 +33,48 @@ const GeneralDetailsAboutLine = () => {
   )
 }
 
+const LineProfileHeader = ({
+  operator_ref,
+  agency_name,
+  route_short_name,
+  route_long_name,
+}: {
+  operator_ref: number
+  agency_name: string
+  route_short_name: string
+  route_long_name: string
+}) => {
+  const { t } = useTranslation()
+  const splitRouteName = route_long_name.split('<->')
+  return (
```

Comment:
You could use this syntax in here : 

`const [firstPart, secondPart] = route_long_name.split('<->');`

A bit cleaner

Reply:
Updated, thanks!

Reply:
How about giving more meaningful names than first and second part?

Can you think of something more indicating?

Reply:
I've  updated it now, thanks again